reg/drone/phy/thermodynamics/PressureTempCovTs.0.1.uavcan

```diff
@@ -0,0 +1,15 @@
+# Timestamped fluid pressure and temperature (sampled synchronously) with covariance.
+# Observe that this is a structural subtype of uavcan.si.sample.pressure.Scalar.1.0.
+
+uavcan.time.SynchronizedTimestamp.1.0 timestamp
+
+uavcan.si.unit.pressure.Scalar.1.0    pressure
+uavcan.si.unit.temperature.Scalar.1.0 temperature
+
+float16[3] covariance
```

Comment:
Actually, this should be split into `pressure_scalar_variance` and `temperature_scalar_variance`. I don't see why we need a covariance of both.

Reply:
Pressure and temperature are tightly correlated, so my assumption is that this information could in theory be leveraged by an external state estimator. The cost of managing a single extra `float16` is low, so I added it. This case is different from, say, kinematic states, because they are known to be uncorrelated.